packages/math-input/src/components/keypad/index.tsx

```diff
@@ -168,6 +168,8 @@ const styles = StyleSheet.create({
         gridTemplateRows: "repeat(4, 1fr)",
         backgroundColor: "#DBDCDD",
         maxHeight: 200,
-        maxWidth: 300,
+        // STOPSHIP we need a maxWidth somewhere for desktop
+        // but can't have it in mobile
+        // maxWidth: 300,
```

Comment:
Would a media query help with this stopship?

Reply:
I think this is a good idea, but several other things change based on the `isMobile` flag in APIOptions and I'd be worried that the media query would eventually get out of sync with `isMobile`. I also tested removing `max-width` in dev tools with Khanmigo and nothing changed, so I think we might not need this at all anymore. 🤔 I'll fiddle around with it, thanks for the suggestion; didn't even think of that.